### ISSUE TITLE
Initialize or fix the transition matrix in CRF layer

### ISSUE THREAD

**Issue by yidatao** [keras-team/keras-contrib#235]
Hi, I'm using keras lstm + crf to solve a NER task. I found that sometimes the crf layer outputs impossible label sequences such as B-Person -> I-Organization. 

To eliminate impossible label transitions, I wanted to manually initialize or fix the transition matrix in crf layer, but couldn't figure out how. Is there any API to do such things?

Here is my code. Thanks for your help.
`model = Sequential()
model.add(Embedding(VOCAB_SIZE, output_dim=EMBEDDING_OUT_DIM, input_length=MAXLEN, mask_zero=True))
model.add(Bidirectional(LSTM(HIDDEN_UNITS, return_sequences=True)))
crf_layer = CRF(NUM_CLASS)
model.add(crf_layer)
model.compile('adam', loss=crf_layer.loss_function, metrics=[crf_layer.accuracy])`

**Comment by BenSNW** [comment]
me too, have you got any idea? @ersinyar I'm considering writing another loss function to regularize those impossible transitions
